src/cwCompassImporter.cpp

```diff
@@ -53,7 +53,7 @@ void cwCompassImporter::runTask()
         parseFile();
 
         //Fix invalid station names
-        StationRenamer.renameInvalidStations();
+//        StationRenamer.renameInvalidStations();
```

Comment:
Commenting this out will probably cause merge conflict with your "change station renaming algorithm #72" commit.